node/network/bridge/src/multiplexer.rs

```diff
@@ -174,6 +194,9 @@ fn multiplex_single(
 		Protocol::StatementFetching => {
 			panic!("Statement fetching requests are handled directly. qed.");
 		}
+		Protocol::DisputeSending => {
+			panic!("Dispute sending request are handled directly. qed.");
```

Comment:
`unreachable!` makes the invariant a bit more explicit, not sure why we don't use it much :monocle_face: